components/member-list-item/index.js

```diff
@@ -1,19 +1,17 @@
-import { string } from 'prop-types';
-import Link from 'next/link';
 import { getDataURL, getImgURL } from 'helper-functions/urls';
+import { useEffect, useRef } from 'react';
```

Comment:
try to keep lib imports on top, followed by custom and local. just a better practice.

Reply:
Not necessarily, I am using a VS code pluggin called "Sort imports" which auto sorts imports on every save with industry standards.

Reply:
`import { getDataURL, getImgURL } from 'helper-functions/urls';
import { useEffect, useRef } from 'react';

import Link from 'next/link';  // then ?
import SocialMediaIcon from '../social-media-icon';
import classNames from './member-list-item.module.scss';
import { string } from 'prop-types';   // then ?
import useFetch from 'custom-hooks/useFetch';`.  // then ?